app.rb

```diff
@@ -1,20 +1,129 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
 apartments = data[:apartments]
 tenants = data[:tenants]
 
+ruby_apartments = []
+ruby_tenants = []
+#id, address, monthly_rent, square_feet
+ruby_apartments.each do |unit|
+  unit = Apartment.new(unit[:id], unit[:address], unit[:monthly_rent], unit[:square_feet])
+  apartments << ruby_apartments
+end
+# id, name, age, apartment_id
+ruby_tenants.each do |tenant|
+  tenant = Tenant.new(tenant[:id], tenant[:name], tenant[:age], tenant [:apartment_id])
+  tenants << ruby_tenants
+end
+puts "1) apartments overview"
+puts "2) tenants overview"
+puts "3) apartments with tenant overview"
+puts "4) find tenant name"
+puts "please enter your choice:"
+
+input = gets.chomp
+
+if input == "1"
+  apartments.map do |unit|
+    puts unit
```

Comment:
Rather than using `puts unit` here, I'd suggest building a nice string like:

```ruby
puts "Apartment at :" + apartment.address + " costs " + apartment.monthly_rent
## OR using string interpolation:
puts "Apartment at : #{apartment.address} costs #{apartment.monthly_rent}"
```